tests/unit_tests/test_deplete_operator.py

```diff
@@ -0,0 +1,72 @@
+"""Basic unit tests for openmc.deplete.Operator instantiation
+
+Modifies and resets environment variable OPENMC_CROSS_SECTIONS
+to a custom file with new depletion_chain node
+"""
+
+from os import remove
+from os import environ
+from unittest import mock
+from pathlib import Path
+import pytest
+
+from openmc.deplete.abc import TransportOperator
+from openmc.deplete.chain import Chain
+
+BARE_XS_FILE = "bare_cross_sections.xml"
+CHAIN_PATH = Path().cwd() / "tests" / "chain_simple.xml"
+
+
+@pytest.fixture(scope="module")
+def bare_xs():
+    """Create a very basic cross_sections file, return simple Chain.
+
+    """
+
+    bare_xs_contents = """<?xml version="1.0"?>
+<cross_sections>
+  <depletion_chain path="{}" />
+</cross_sections>
+""".format(CHAIN_PATH)
+
+    with open(BARE_XS_FILE, "w") as out:
+        out.write(bare_xs_contents)
+
+    yield
+    remove(BARE_XS_FILE)
```

Comment:
There are two options in place for using a temporary directory that you can use instead of manually removing this:
1. In `tests/conftest.py`, there is a fixture called `run_in_tmpdir` that will run a test in a temporary directory and then switch back to wherever the cwd was before.
2. In `tests/__init__.py`, there is a `cdtemp` function that can be using as a context manager to temporary change into a temporary directory.

Reply:
Will use the run_in_tmpdir option